tycho-its/projects/tycho.xtend/pom.xml

```diff
@@ -0,0 +1,49 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>tycho.xtend</groupId>
+	<version>1.0.0-SNAPSHOT</version>
+	<artifactId>tycho.xtend</artifactId>
+	<packaging>eclipse-plugin</packaging>
+	
+	<properties>
```

Comment:
dont know what to keep in this pom and what to sort out

Reply:
I think this is all fine.